dotnetv3/cross-service/AuroraItemTracker/Readme.md

```diff
@@ -0,0 +1,180 @@
+    --database "DATABASE" ^
+    --secret-arn "SECRET_ARN" ^
+    --sql "CREATE TABLE items (id VARCHAR(45), description VARCHAR(400), guide VARCHAR(45), status VARCHAR(400), name VARCHAR(45), archived BOOLEAN);"
+```
+
+### Verified email address
+
+To email reports from the app, you must register at least one email address with Amazon SES. This verified email is specified as the sender for emailed reports.
+
+1. In a browser, navigate to the Amazon SES console.
+1. If necessary, select your AWS Region.
+1. Select Verified identities.
+1. Select Create identity.
+1. Select Email address.
+1. Enter an email address you own.
+1. Select Create identity.
+1. You will receive an email from Amazon Web Services that contains instructions on how to verify the email with Amazon SES. Follow the instructions in the email to complete verification.
+
+Tip: For this example, you can use the same email account for both the sender and the recipient.
+
+## Build the code
+
+This application has two parts: a user interface that uses React.js and a
+RESTful API created with C# and .NET 6. The React user interface is a single-page
+application (SPA) that interacts with the C# RESTful API by making `GET`, `PUT`, and
+`POST` requests.
+
+### RESTful API
+
+The API uses the [AmazonRDSDataServiceClient](https://docs.aws.amazon.com/sdkfornet/v3/apidocs/items/RDSDataService/TRDSDataServiceClient.html)
+object to perform CRUD operations on an Aurora Serverless database. The API
+returns JSON data in an HTTP response, as shown in the following illustration.
+
+![AWS Tracker JSON response](images/aurora_item_tracker_response.png)
+
+#### Configure the application
+Before you run the .NET application, set the configuration values for your Aurora serverless
+database, and your verified email address in the `appsettings.json` file. Alternatively, add an `appsettings.development.json` file
+with your local settings.
+
+#### Run the application
+After the example compiles, you can run it from the command line. To do so,
+navigate to the folder that contains the .csproj file and run the following
+command:
+
+```
+dotnet run
+```
+
+Alternatively, you can run the example from within your IDE.
+
+#### Tests
+
+⚠️ Running the tests might result in charges to your AWS account.
+
+The solution includes a test project. To run the tests, navigate to the folder that contains the test project and then issue the following command:
+
+```
+dotnet test
+```
+
+Alternatively, you can open the example solution and use the Visual Studio Test Runner to run the tests.
+
+### Aurora Item Tracker user interface
+To start the React web application, you can download files from the following GitHub repository. Included in this repository are instructions on how to set up the project. Click the following link to access the GitHub location [Item Tracker web client](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/resources/clients/react/item-tracker/README.md).  
+
```

Comment:
This is confusing, and makes it sound like you have to get the client from a different GitHub repository, when it's all in the current repo that you already have. I would just discuss the client and provide the pointer to the README for Elwing.

Reply:
Cleaned up this paragraph to mostly just refer to the README for the web client.